src/S1_fft_cc_MPI.py

```diff
@@ -294,7 +295,9 @@
 
             # load fft data in memory for cross-correlations
             data = source_white[:,:Nfft2]
-            fft_array[iii] = data.reshape(data.size)
+            data= data.reshape(1, data.size)
+            filler=np.empty((1,data.size))
+            fft_array[iii] = np.concatenate((data,filler),axis=1)
```

Comment:
@niyiyu @mdenolle 
At this point `fft_array[iii].shape = (72000,)`, but `data.shape = (36000,)`, so the assignment fails. Padding with 36000 `0s` works, ie avoids the crash, but is this the right thing to do? Did this code used to work?
I wonder if could be a mismatch of parameters since these dimensions are derived out of params such as `cc_len, step, samp_freq`, etc.